netmanager/src/views/stateConnectors/containerConnector.js

```diff
@@ -0,0 +1,49 @@
+// map state from store to props
```

Comment:
This is a good refactor, the usage of descriptive names for all of our files/directories is extremely welcome.

Reply:
Coming up with good names takes sometime. I prefer to do it as last thing 😄

Reply:
@Baalmart This is fixed